### DISCUSSION TITLE
Garbo trying to drink wrecked generator for aftercore/nightcap (wineglass)

### DISCUSSION THREAD

**Discussion by dutchladymilk** [loathers/garbage-collector#1917] [Q&A] [answered]
Is there a way to make it drink something else besides wrecked generators? Been happening for the past week or so.

**Comment by pstalcup** [comment]
Why is this a problem?

**Reply by dutchladymilk** [reply]
wrecked generator requires one to finish the questline before being accessible, which is a slight problem if one where to ascend daily. In my case since it cannot be acquired, garbo aborts

**Reply by pstalcup** [reply]
as far as I can tell, Garbo should not consider wrecked generators at all. Is it possible its another script that it is trying to consume a wrecked generator?

**Reply by dutchladymilk** [reply]
I'll look into it. Thank you!

**Reply by dutchladymilk** [reply]
I think CONSUME is the problem

**Reply by pstalcup** [reply, marked as answer]
https://github.com/soolar/CONSUME.ash would be the place to make an issue. garbo does not invoke consume at all.
